pkg/scheduler/actions/backfill/backfill.go

```diff
@@ -68,15 +68,15 @@ func (backfill *Action) Execute(ssn *framework.Session) {
 		fe := api.NewFitErrors()
 
 		if err := ssn.PrePredicateFn(task); err != nil {
-			klog.V(3).Infof("PrePredicate for task %s/%s failed in backfill for: %v", task.Namespace, task.Name, err)
+			klog.Errorf("PrePredicate for task %s/%s failed in backfill for: %v", task.Namespace, task.Name, err)
```

Comment:
we'd better use level 4. Otherwise there will be a lot of error and this kind error can be ignored.

Reply:
done